src/random.jl

```diff
@@ -6,13 +6,13 @@ Returns the connectance of a square matrix, defined as ``S/L^2``.
 """
 function connectance(S::Int64, L::Int64)
     @assert S > 1
-    C = L/S^2
+    C = L / S^2
```

Comment:
Cosmetic changes belong to #2. (no offense, uh? I'm just testing github's reviewing interface ;)

Reply:
Wops, apparently the cosmetic changes surived your rebase. So this is not resolved yet ;)